bundle/config/bundle.go

```diff
@@ -28,4 +28,12 @@ type Bundle struct {
 	// Contains Git information like current commit, current branch and
 	// origin url. Automatically loaded by reading .git directory if not specified
 	Git Git `json:"git,omitempty"`
+
+	// Determines the mode of the environment.
+	// For example, 'mode: development' can be used for deployments for
+	// development purposes.
+	Mode Mode `json:"mode,omitempty"`
+
+	// Overrides the compute used for jobs and other supported assets.
+	Compute string `json:"override_compute,omitempty" bundle:"readonly"`
```

Comment:
Why use a different name for the JSON field and the Go field?

Does the compute field in the jobs API apply to clusters as well? Or do you take this value and plan to use it in both the compute and cluster fields in the job specs?

Reply:
Also, it is used as ID, so should be called `ComputeID` to match.

Reply:
What is the expected behavior if this value is set by the user?

Reply:
I updated this to just say computeId/ComputeID. Is "-id" the customer-facing convention we have throughout the spec?

Note that this wasn't meant to set from the config, hence the "readonly," but that doesn't seem to do what I expected it to. I updated it so it can be set at the environment level now.x

In terms of naming, what I can do is also change the parameter name to say "--compute-id"/"-c" instead of "--compute"? It's a bit longer but I tentatively made that change.